### PR TITLE
Add _pytest.fixtures.FixtureLookupError to top level import

### PR DESCRIPTION
**Pull request by earonesty**
TODO:

- [X] Include new tests or update existing tests
- [X] Create a new changelog file in the `changelog` folder, with a name like `<ISSUE NUMBER>.<TYPE>.rst`. See [changelog/README.rst](https://github.com/pytest-dev/pytest/blob/master/changelog/README.rst) for details.
- [X] Add yourself to `AUTHORS` in alphabetical order.

SEE #6280

### CODE REVIEW

File: src/pytest.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 from _pytest.fixtures import fillfixtures as _fillfuncargs
 from _pytest.fixtures import fixture
 from _pytest.fixtures import yield_fixture
+from _pytest.fixtures import FixtureLookupError
```
Comment (RonnyPfannschmidt) [inline src/pytest.py]: please add this one to __all__

File: changelog/6285.feature.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Use `pytest.FixtureLookupError` to catch exceptions when calling `request.getfixturevalue`.
```
Comment (bluetech) [inline changelog/6285.feature.rst]:
```suggestion
Exposed the `pytest.FixtureLookupError` exception which is raised by `request.getfixturevalue()`
(where `request` is a `FixtureRequest` fixture) when a fixture with the given name cannot be returned.
```

**Comment by bluetech** [comment]
This also needs a rebase unfortunately!

**Comment by nicoddemus** [comment]
> It would be good to add a note to FixtureRequest.getfixturevalue's docstring that it raises FixtureLookupError if a fixture with this name cannot be returned.

Definitely, done. 👍 

> This also needs a rebase unfortunately!

Done. 👍

**Comment by nicoddemus** [comment]
We should definitely squash/merge this.

Thanks again @earonesty.

**Comment by RonnyPfannschmidt** [comment]
i just noted this one while looking at our oldest pr's - sorry it took so long

**Pull request merged**